### ISSUE TITLE
feature request: run the script unsupervised 

### ISSUE THREAD

**Issue by acc0j** [k3karthic/bitwarden-to-keepass#1]
Hi, 

I'm looking to set up a daily export from Bitwarden to a kdbx file, for which your scripts seems to be good tool. However, for the daily job I have in mind, there are a few modifications required. I'd like to check with you how you prefer this: (a) I can fork your repo and just do my modifications, you don't need to be bothered by it; (b) I can send you a PR with some modifications; (c) I can let you know my feature request and you have a look.

This are the modifications I'm thinking about:
1) Get the master password from an environment variable, and only request user input if the environment variable is not set.
2) Currently the script crashes if it finds two items in the same folder with the same name. Some better exception handling of this would be nice, i.e. appending "copy1" or something to the name in case this happens.

Kind regards,

**Comment by k3karthic** [comment]
Hi,

Thanks for the feedback. I've pushed an update which will allow you to specify the password in the environment variable `BITWARDEN_PASS`.

I tried creating two entries with the same name but the script does not throw an error. Could you send me a screenshot of the error you're getting? Please feel free to send a PR if that's easier for you.

OS: Debian Buster
Python: v3.7.3
pykeepass: v4.0.1
BitWarden CLI: v1.19.1

![image](https://user-images.githubusercontent.com/4071529/142766865-d8764311-01b3-4cdd-8845-f72e88769618.png)

Regards,

**Comment by acc0j** [comment]
Thanks k3karthic for the quick response on the environment variable!

About the two entries, let me show you; this is my Bitwarden vault:
![afbeelding](https://user-images.githubusercontent.com/21264520/142768449-55756761-db35-4023-9882-8fa650edec48.png)

It gives this output:
```
Traceback (most recent call last):
  File "/app/convert.py", line 133, in <module>
    convert(args.Output)
  File "/app/convert.py", line 114, in convert

Fetching folders...
Fetching items...

    kp.add_entry(
  File "/usr/local/lib/python3.9/site-packages/pykeepass/pykeepass.py", line 613, in add_entry
    raise Exception(
Exception: An entry "EntryForTest" already exists in "Group: """
```
When I remove one of the entries, it works as expected.

Running in a Docker container with:
* Debian Buster
* Python 3.9
* pykeepass 4.0.1
* BW CLI 1.19.1

**Comment by acc0j** [comment]
Hi,

Some more insights : it appears to only occur if both the name of the entry and the usernames are identical. So it is possibly a relatively rare use care. Still it might be more interesting to output a kdbxfile without the conflicting entries (and showing a warning or error) than no file at all. Or possibly let the user choose the behaviour with e.g. a --fail-on-error=true/false flag or similar. I'll leave it up to you to see if and how you want to handle this.

For my part, I renamed the conflicting entries so no problem anymore.

**Comment by k3karthic** [comment]
Hi vojeroen,

Thank you for taking the time to look into this. `convert.py` is a backup script and I would like to avoid dropping credentials without warning.

I like your idea of resolving the conflict by appending a sequential suffix like "copy 1". I'll implement it sometime this week.

Regards,

**Comment by k3karthic** [comment]
I've released v0.1.3 which adds a suffix like “\<name\> (1)” to the title for duplicate entries.

**Comment by acc0j** [comment]
I'll give it a go, thanks!
